Simplenote/LoginWindowController.h

```diff
@@ -17,6 +18,8 @@
  *              extra functionality, Simplenote-Y.
  */
 
-@interface LoginWindowController : SPAuthenticationWindowController
+@interface LoginWindowController : SPAuthenticationWindowController <NSWindowDelegate> {
+    WPAuthWindowController *wpAuthWindowController;
```

Comment:
The "latest" ObjC syntax would look somehow like this:

```
@property (nonatomic, strong) WPAuthWindowController *wpAuthWindowController;
```

That automatically generates the ivar and synthesized setter / getter. (Nevertheless, this snippet is also valid as well!!)